src/lib/config.js

```diff
@@ -181,12 +240,13 @@ function getLocalConfig () {
     admin,
     tradingPairs,
     server,
-    backendUri
+    backendUri,
+    notifications
   }
 }
 
 function loadConnectorConfig () {
-  return Config.loadConfig(envPrefix, getLocalConfig())
+  return Config.loadConfig(envPrefix, getLocalConfig(), {ed25519: false})
```

Comment:
The config module in five-bells-shared currently tries to parse ed25519 keys for all components. In production it fails if they don't exist. This is a workaround to avoid releasing a breaking change in five-bells-shared right now. I have a followup to revisit this in the next sprint (RILP-409)